News_Blockchain/BlockValidator.cs

```diff
@@ -492,5 +505,18 @@ public static Block CreateBlock(List<Transaction> transactions)
 
             return block;
         }
+
+        /// <summary>
+        /// Function creates valid transaction
+        /// </summary>
+        /// <param name="address">Recipient's address</param>
+        /// <param name="value">Value of transfer</param>
+        /// <param name="fee">Fee in satoshy/byte</param>
+        /// <param name="text">Optional argument for news content</param>
+        /// <returns></returns>
+        public static Transaction CrateTransaction(string address, double value, int fee, string text = "")
```

Comment:
@Luka1321 Try writing this function on your own. All of the needed functionality has already been written, you just need to put Legos together